src/responder/common/cache_req/cache_req_private.h

```diff
@@ -103,6 +103,7 @@ struct cache_req_data {
 
     /* if set, only search in the listed domains */
     char **requested_domains;
+    bool return_offline;
```

Comment:
IMO, this field name isn't very suitable / telling.

Reply:
Can you suggest something better?

Reply:
`There are 2 hard problems in computer science: cache invalidation, naming things, and off-by-1 errors.` :)

Maybe `propogate_offline_status`?

But perhaps it's enough to leave an explanatory comment (as it's done for previous field).

Reply:
This is better, done.